src/screens/Home.tsx

```diff
@@ -1,107 +1,196 @@
+        Inter_700Bold          // Load the 'Inter_700Bold' font style
+    });
+    //Check if fonts are loaded 
       if (!fontsLoaded) {
-        // You can return a loading screen or a placeholder here until the font is loaded
-        return <View style={styles.body}>
+        //If not loaded return a loading screen
+        return (
+          <view style={styles.body}>
+            {/* <!-- apply the default gradient as a background --> */}
             <DefaultGradient />
-            <Text style={styles.buttontext}>Loading...</Text>
-            </View>;
+            {/* <!-- display a "loading..." text with a specific style --> */}
+            <text style={styles.buttontext}>loading...</text>
+          </view>
+        );
       }
-    return (
+    // If fonts are loaded, return the following
+      return (
+        // Begin a container for the main body
         <View style={styles.body}>
+            {/* <!-- Apply the default gradient as a background --> */}
             <DefaultGradient />
+            {/* <!-- Begin a nested container for the app's main content --> */}
             <View style={styles.container}>
-            <Text style={styles.text}>Car Rental App</Text>
-            <Pressable style={styles.cityselect}>
-                <Text style={styles.cityText}>Select City</Text>
-            </Pressable>
-            <StatusBar style="auto" />
-            <Pressable
-                style={styles.button}
-                onPress={() => navigation.push('CarBrowser')}
-            >
-                <View style={styles.row}>
-                    <Text style={styles.buttontext}>GO! </Text>
-                    <Image source={require('../../assets/GO.png')} style={{ width: imageSize, height: imageSize }} />
-                </View>
-            </Pressable>
+                {/* <!-- Display the app's title "Car Rental App" with a specific style --> */}
+                <Text style={styles.text}>Car Rental App</Text>
+                {/* <!-- Create a clickable (pressable) button for city selection --> */}
+                <Pressable style={styles.cityselect}>
+                    {/* <!-- Display the "Select City" text within the button with a specific style --> */}
+                    <Text style={styles.cityText}>Select City</Text>
+                </Pressable>
+                {/* <!-- Display the device's status bar with an automatic style --> */}
+                <StatusBar style="auto" />
+                {/* <!-- Create a clickable (pressable) button that will navigate to the 'CarBrowser' screen when pressed --> */}
+                <Pressable
+                    style={styles.button}
+                    onPress={() => navigation.push('CarBrowser')}
+                >
+                    {/* <!-- Begin a container for the button's content which includes text and an image --> */}
+                    <View style={styles.row}>
+                        {/* <!-- Display the "GO!" text with a specific style --> */}
```

Comment:
Come on. make comments where it makes sense.

A Text element with a text and style is pretty self-explanatory.

Reply:
Pressable is a weird React native component, that makes everything inside of it pressable with an action. Thought a comment would clarify this abstract component